### DISCUSSION TITLE
The icons

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [JanDeDobbeleer/oh-my-posh#1222] [Troubleshoot]
Where can i find the icons? Because the ocons for the time is offset... It is down around 1-5 pixels and i wanna change it. How and where can I find it?

**Comment by JanDeDobbeleer** [comment]
What do you mean?
- I need to find al alternative icon -> [Here](https://www.nerdfonts.com/cheat-sheet)
- I have no idea how to adjust -> [Create you own theme](https://ohmyposh.dev/docs/windows#customize)
- I don't know where these settings are -> [Have a look at the docs](https://ohmyposh.dev/docs/time)
